core/dictgen/src/rootcling_impl.cxx

```diff
@@ -207,6 +215,19 @@ const char *GetExePath()
       exepath = buf;
     }
 #endif
+#if defined R__FBSD || defined __FreeBSD__
```

Comment:
I think we should be consistent on the use of `defined(R__FBSD)` and `defined(__FreeBSD__)`.  IMHO, we should stick to `R__FBSD` except perhaps in cling, which can be build standalone, where we should use `defined(__FreeBSD__)` instead.
```suggestion
#if defined(R__FBSD)
```